personnel.md

```diff
@@ -20,15 +20,15 @@ Feel free to contact the secretaries at info AT php-fig.org. For more informatio
 | Larry Garfield ([@Crell@phpc.social])             | 2016-12-24 - 2025-05-30 |
 | Matthew Weier O'Phinney ([@mwop])                 | 2016-12-24 - 2025-05-30 |
 | Cees-Jan Kiewiet ([@wyri@haxim.us])               | 2016-12-24 - 2024-08-31 |
-| Chris Tankersley ([@dragonmantank])               | 2016-12-24 - 2024-01-28 |
-| Korvin Szanto ([@korvinszanto])                   | 2016-12-24 - 2024-01-28 |
+| Korvin Szanto ([@korvinszanto])                   | 2016-12-24 - 2026-02-22 |
 | Chuck Burgess ([@ashnazg@phpc.social])            | 2018-08-26 - 2024-08-31 |
-| Enrico Zimuel ([@ezimuel])                        | 2020-01-25 - 2024-01-28 |
 | Alessandro Chitolina ([@alekitto@phpc.social])    | 2021-06-25 - 2025-05-30 |
 | Ken Guest ([@kenguest@phpc.social])               | 2022-01-30 - 2024-01-28 |
 | Jaap van Otterdijk ([@jaapio@phpc.social])        | 2022-08-28 - 2024-08-28 |
 | Navarr Barnier ([@navarr])                        | 2022-08-28 - 2024-08-28 |
 | Vincent de Lau ([@vdelau])                        | 2023-05-25 - 2025-05-30 |
+| Steve McDougall                                   | 2024-02-25 - 2026-02-22 |
+| Matteo Beccati ([@mbeccati])                      | 2024-02-25 - 2026-02-22 |
```

Comment:
I'm not checking twitter any longer. Could you please change to `@mbeccati@phpc.social`? Thank you in advance!

Reply:
@mbeccati I have updated your contact information, thanks for providing it.

Reply:
@mbeccati please re-review, you have Changes Requested on this, which is blocking the PR. Changes have been addressed. Thanks!

Reply:
@mbniebergall I thought resolving the thread was enough, sorry. I've approved the PR, but it looks like it's still blocked waiting for Secretaries' review.